src/SFA.DAS.Notifications.Api/Web.Release.config

```diff
@@ -1,19 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
-
-  <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.nlog-project.org/schemas/NLog.xsd NLog.xsd" autoReload="true" throwExceptions="true" xdt:Transform="RemoveAttributes(internalLogLevel,internalLogFile)">
-    <rules>
-      <logger name="*" minlevel="Info" writeTo="EventHubBuffered" xdt:Transform="SetAttributes" xdt:Locator="Match(name)" />
-    </rules>
-  </nlog>
-
   <system.web>
     <compilation xdt:Transform="RemoveAttributes(debug)" />
   </system.web>
   <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.nlog-project.org/schemas/NLog.xsd NLog.xsd" autoReload="true" throwExceptions="true" xdt:Transform="RemoveAttributes(internalLogLevel,internalLogFile)">
     <rules>
       <logger minlevel="Debug" writeTo="RedisLog" xdt:Transform="SetAttributes" />
+      <logger name="*" minlevel="Trace" writeTo="aiTarget" xdt:Transform="Insert"/>
```

Comment:
I'd say both, it does give us more options but mainly it can help to give context to any issues highlighted by application insights